app/portainer/services/codeMirror.js

```diff
@@ -12,13 +17,17 @@ angular.module('portainer.app').factory('CodeMirrorService', function CodeMirror
 
   var codeMirrorGenericOptions = {
     lineNumbers: true,
+    extraKeys: {
+      "Alt-F": "findPersistent"
+    }
```

Comment:
I had a play around with this search feature and it seems to work without the extraKeys. All the keys get bound automatically just by importing the addons

Maybe we can just remove these

Reply:
Thank you for the review.

The declaration of this extrakey (alt+f) is for the find that stay even if you click somewhere else.
If not specified, alt+f will open the settings menu instead.
Tested today with chrome 103.0.5060.114 and Firefox 102.0.1 on Ubuntu 22.04